src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/configurationtechnologies/DotNetOptions.java

```diff
@@ -0,0 +1,45 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies;
+
+import hudson.Extension;
+import hudson.model.AbstractProject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.export.Exported;
+
+public class DotNetOptions /*extends TechnologyOptions*/ {
```

Comment:
we don't use DotNet option right now so we comment out the inheritance so its wont show on ui